PEPit/point.py

```diff
@@ -298,6 +301,23 @@ def eval(self):
 
         return self._value
 
+    def eval_ld(self):
+        """
+        Evaluation in lower dimension.
+
+        Returns:
+            value (np.array): The trimmed value of this :class:`Point` after the corresponding PEP was solved numerically.
+
+        Raises:
+            ValueError("The PEP must be solved to evaluate Points!") if the PEP has not been solved yet.
+
+        """
+        if Point._toggled_dimensions is not None:
+            toggled_dimensions = Point._toggled_dimensions
+        else:
+            toggled_dimensions = Point.counter
+        
+        return self.eval()[:toggled_dimensions]
```

Comment:
Since toggled_dimensions can be handily fixed, this can modify the inner product values. Shall we consider that the user is responsible for doing the right thing? Or should we enforce toggled_dimensions to be automatically fixed?